main.js

```diff
@@ -13,6 +13,275 @@ toggle.addEventListener('click', () => {
 
   navlink.classList.toggle('active');
 });
+const cardsData = [
+  {
+    class: 'ttl',
+    class2: 'cards',
+    heading: 'My Recent Works',
+  },
+  {
+    class: ['img-hdr', 'langs'],
+    divCls: [
+      'card cd1',
+      'card cd2',
+      'card cd3',
+      'card cd4',
+      'card cd5',
+      'card cd6',
+    ],
+    listcls: 'cd-links',
+    heading: ['Multi-Post Stories', 'Gain+Glory'],
+    lists: ['Ruby on Rails', 'css', 'Javascript', 'html'],
+    btn: 'See Project',
+    btnCls: 'cardBtn',
+  },
+];
+const popupData = [
+  {
+    id: 'popup',
+    divCls: ['popimg', 'poptext', 'popHead', 'topBtns', 'btns'],
+    classes: [
+      'fa-solid fa-x close',
+      'imgMob',
+      'imgDesk',
+      'popTtl',
+      'btn seeLive',
+      'btn seeLink',
+      'popLinks',
+      'link',
```

Comment:
- [ ] I must commend your tenacity to push this project to this point. Well done 👏🏾. However, I see that you popup is not properly align in the center on the desktop view. Also, it is not scrolling down to make visible the text hidden. See image below to see your work in camparison to the Figma template.

Your work
![image](https://user-images.githubusercontent.com/84156714/218053405-01b81d7b-d643-40ba-beef-c64fc17c54d4.png)

Figma Template
![image](https://user-images.githubusercontent.com/84156714/218053920-552ab23d-ee53-4e0a-8d06-16f8dbe3b3ad.png)

You should try as much as possible to duplicate the Figma Template.